pkg/pod/status.go

```diff
@@ -114,7 +115,6 @@ func MakeTaskRunStatus(tr v1alpha1.TaskRun, pod *corev1.Pod, taskSpec v1alpha1.T
 	trs.PodName = pod.Name
 	trs.Steps = []v1alpha1.StepState{}
 	trs.Sidecars = []v1alpha1.SidecarState{}
-	logger, _ := logging.NewLogger("", "status")
 	defer func() {
 		_ = logger.Sync()
```

Comment:
I have no idea if this needs to stick around or not now that we're passing in the logger from the controller.